src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,37 @@
-export const Tabs = () => {};
+export const Tabs = ({ tabs, selectedTab, onTabSelected }) => {
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map(tab => {
+            const isTabSelected = selectedTab.id === tab.id;
+
+            return (
+              <li
+                key={tab.id}
+                className={isTabSelected ? 'is-active' : ''}
```

Comment:
It's better to use `classnames` library to set classes conditionally https://www.npmjs.com/package/classnames